tower-oauth2-resource-server/src/authorizer/jwt_validate.rs

```diff
@@ -85,49 +92,33 @@ impl OnlyJwtValidator {
     }
 }
 
-impl OnlyJwtValidatorInner {
-    fn update_validations(&mut self, jwks: &JwkSet, claims_validation: &ClaimsValidationSpec) {
-        let algs = jwks
-            .keys
-            .iter()
-            .filter_map(|jwk| jwk.common.key_algorithm)
-            .filter_map(parse_key_alg)
-            .collect::<HashSet<_>>();
-        self.validations = algs
-            .into_iter()
-            .map(|alg| (alg, self.create_validation(&alg, claims_validation)))
-            .collect();
-    }
-
-    fn create_validation(
-        &self,
-        alg: &Algorithm,
-        claims_validation: &ClaimsValidationSpec,
-    ) -> Validation {
-        let mut validation = Validation::new(*alg);
-        let mut required_claims = Vec::<&'static str>::new();
-        if let Some(iss) = &claims_validation.iss {
-            required_claims.push("iss");
-            validation.set_issuer(&[iss]);
-        }
-        if claims_validation.exp {
-            required_claims.push("exp");
-            validation.validate_exp = true;
-        }
-        if claims_validation.nbf {
-            required_claims.push("nbf");
-            validation.validate_nbf = true;
-        }
-        if !claims_validation.aud.is_empty() {
-            required_claims.push("aud");
-            validation.set_audience(&claims_validation.aud);
-        } else {
-            validation.validate_aud = false;
-        }
-        validation.set_required_spec_claims(&required_claims);
-        validation
+/// Creates a validation configuration for the given algorithm and claims validation spec.
```

Comment:
I think this comment basically say the same thing as the code that it refers to.
Do you mind removing it? :)